spec/game_spec.rb

```diff
@@ -14,87 +14,114 @@
       game = Game.new(board, display, player1, player2)
 
       allow($stdin).to receive(:gets).and_return('1')
-      expect do
-        game.play_move
-      end
-      .to output(
-      """
+      expect { game.play_move }
+        .to output("""
      1 | 2 | 3
     -----------
      4 | 5 | 6
     -----------
      7 | 8 | 9\nThe current player is x\nChoose a position from 1-9: """)
-    .to_stdout
+        .to_stdout
     end
   end
 
-  context 'Playing a move when x is the current player' do
+  context 'Playing a move when x is the current player:' do
     display = Display.new
     player1 = Player.new('x')
     player2 = Player.new('o')
     board = Board.new([1, 2, 3, 4, 5, 6, 7, 8, 9])
     game = Game.new(board, display, player1, player2)
 
-    it 'plays move of x and updates current player to o' do
+    it 'plays move of x and updates the current player to o' do
       allow($stdin).to receive(:gets).and_return('1')
+
       game.play_move
+
       expect(game.current_player.mark).to eq('o')
     end
 
-    it 'plays move of o and updates current player to x' do
+    it 'plays move of o and updates the current player to x' do
       allow($stdin).to receive(:gets).and_return('9')
+  
       game.play_move
+
       expect(game.current_player.mark).to eq('x')
     end
   end
 
-  context 'Checks if user can continue playing' do
+  context 'Checks if a user can continue playing:' do
     display = Display.new
     player1 = Player.new('x')
     player2 = Player.new('o')
 
-    it 'when less than 9 moves have been played, can_continue_playing returns true' do
-      allow($stdin).to receive(:gets).and_return('9')
+    it 'returns false when less than 9 moves have been played' do
       board = Board.new([1, 'x', 'x', 'o', 5, 6, 'o', 8, 9])
       game = Game.new(board, display, player1, player2)
-      game.play_move
-      expect(game.can_continue_playing?).to eq(true)
+
+      expect(game.is_over?).to eq(false)
     end
 
-    it 'when 9 moves have been played, can_continue_playing returns false' do
-      allow($stdin).to receive(:gets).and_return('9')
-      board = Board.new(['o', 'x', 'x', 'o', 'x', 'o', 'x', 'o', 9])
+    it 'returns true when 9 moves have been played' do
+      board = Board.new(['o', 'x', 'x', 'o', 'x', 'o', 'x', 'o', 'x'])
       game = Game.new(board, display, player1, player2)
-      game.play_move
-      expect(game.can_continue_playing?).to eq(false)
+
+      expect(game.is_over?).to eq(true)
```

Comment:
prefer `be_true`

Reply:
Thanks, Matt. I need to get familiar with the different matchers in RSpec